transports/http/src/main/java/org/mule/transport/http/components/StaticResourceMessageProcessor.java

```diff
@@ -61,7 +62,24 @@ public MuleEvent process(MuleEvent event) throws MuleException
         }
 
         String path = event.getMessage().getInboundProperty(HttpConnector.HTTP_REQUEST_PATH_PROPERTY);
-        String contextPath = event.getMessage().getInboundProperty(HttpConnector.HTTP_CONTEXT_PATH_PROPERTY);
+        String contextPath = event.getMessage().getInboundProperty(org.mule.module.http.api.HttpConstants.RequestProperties.HTTP_LISTENER_PATH);
```

Comment:
Is http transprot still supported? We should continue to support it until it's removed but I'm unsure if this change does this.